modules/dnn/src/int8layers/quantization_utils.cpp

```diff
@@ -41,7 +41,7 @@ static void broadcast1D2TargetMat(Mat& data, const MatShape& targetShape, int ax
 static void block_repeat(InputArray src, const MatShape& srcShape, int axis, int repetitions, OutputArray dst)
 {
     CV_Assert(src.getObj() != dst.getObj());
-    CV_Check(axis, axis >= 0 && axis < src.dims(), "Axis out of range");
+    CV_Check(axis, axis >= 0 && (axis < src.dims() || (src.dims()==1 && axis==1)), "axis is out of range");
```

Comment:
> src.dims()==1 && axis==1

It does not look valid to me for the general cases.

Reply:
@fengyuentau, yes, this is hack, because the generated model contains incorrect, out-of-range axis specification. The script needs to be fixed and then dexined model should be regenerated

Reply:
Ok, then @gursimarsingh could you add a comment for this?